tasks/main.yml

```diff
@@ -4,24 +4,39 @@
   when: "libvirt_vms | rejectattr('state', 'eq', 'absent')"
 
 - include_tasks: volumes.yml
+  vars:
+    libvirt_vm_volumes: "{{ vm.volumes | default([]) }}"
   with_items: "{{ libvirt_vms }}"
   loop_control:
     loop_var: vm
   when: (vm.state | default('present')) == 'present'
 
 - include_tasks: vm.yml
+  vars:
+    libvirt_vm_console_log_path: "{{ vm.console_log_path | default(libvirt_vm_default_console_log_dir + '/' +  vm.name + 'console.log') }}"
```

Comment:
Does this need a hyphen before console.log?

Reply:
Yep.